faq/dedibox-in-scaleway-console.mdx

```diff
@@ -35,10 +45,6 @@ The account linking process takes about 5 minutes. This is the time required to
 If you already have a Scaleway account, you must be the Owner of the [Organization](/identity-and-access-management/iam/concepts/#organization) you would like to link your Dedibox account to.
 You should also make sure that this Organization has the same country as your Dedibox account.  If it's not the case, you should create a new account, or [contact our support team](/console/my-account/how-to/open-a-support-ticket/) to change the country.
 
-## Are my other Online.net services (web hosting, domains, Dedirack, etc.) linked to my Scaleway account during the process?
-
-No. The other Online.net services will not be linked at this stage but will remain accessible in the Dedibox console.
-
 ## Is there date when the Dedibox console (console.online.net) will stop being available?
 
 No. The current Dedibox console will remain accessible.
```

Comment:
```suggestion
The current Dedibox console will remain accessible.
```

Reply:
Maybe you could add details as to what you will still be able to do on the Dedibox console vs what you won't be able to do.